controllers/volumereplicationgroup_controller.go

```diff
@@ -1007,15 +1007,17 @@ func (v *VRGInstance) err(err error, msg string, requeue bool,
 	msg = fmt.Sprintf("%s: %v", msg, err)
 	conditionSet(&v.instance.Status.Conditions, v.instance.Generation, msg)
 
-	return v.updateVRGStatus(ctrl.Result{Requeue: requeue}, false)
+	return v.updateVRGStatus(ctrl.Result{Requeue: requeue})
```

Comment:
Shouldn't this be `updateVRGConditionsAndStatus`

Reply:
The `false` parameter indicates the conditions update routine should not be called; it is only called after the reconcile as primary or secondary routines.